driver/driver.go

```diff
@@ -118,7 +132,46 @@ func NewDriver(url, endpoint, nodeID, account string, mounter *mount.SafeFormatA
 		nodeCaps: []csi.NodeServiceCapability_RPC_Type{
 			csi.NodeServiceCapability_RPC_STAGE_UNSTAGE_VOLUME,
 		},
-	}, nil
+		attacher: attacher,
+		quit:     make(chan bool),
+	}
+
+	driver.log.Info("Starting JWT maintainer to refresh the JWT at least once each 30 days.")
+	driver.client.JWT.Get()
+	driver.jwtRefresher = time.NewTimer(29 * 24 * time.Hour)
```

Comment:
I think a timer will send an event to the channel once, I think you need to use `time.Ticker` for this.
https://golang.org/pkg/time/#Ticker
https://golang.org/pkg/time/#Timer